scripts/config/templates/nginx.conf.erb

```diff
@@ -126,6 +126,16 @@ http {
     return 301 https://$host/search-q;
   }
 
+  # This will allow us to replicate cloudflare 502 errors we are seeing in search.
+  # Once the issue is resolved, this route should be removed from here and the ssr-frontend.
+  location = /cloudflare-error-playground {
+    proxy_pass <%= ssr_frontend_host %>/cloudflare-error-playground$is_args$args;
+
+    proxy_buffer_size 128k;
+    proxy_buffers 16 128k;
+    proxy_busy_buffers_size 256k;
```

Comment:
Why do you need this buffer changes?

Reply:
This is just to match search. Want to get as similar of an experience as possible.